apps/CA.pl.in

```diff
@@ -209,7 +209,9 @@ if ($WHAT eq '-newcert' ) {
 } elsif ($WHAT eq '-verify' ) {
     my @files = @ARGV ? @ARGV : ( $NEWCERT );
     foreach my $file (@files) {
-        my $status = run("$VERIFY -CAfile ${CATOP}/$CACERT $file $EXTRA{verify}");
+        # -CAfile quoted for VMS, since the C RTL downcases all unquoted
+        # arguments to C programs
```

Comment:
Another reason to make them case insensitive :)

Reply:
Yeah...  that's not very Unix, though :wink:

Reply:
Nah, with long options you don't run out so using upper and lower case single character options isn't required any more.  Still, shell meta characters were fun as options.

Reply:
maybe make options *on VMS* case-insensitive.